main.rb

```diff
@@ -22,7 +23,7 @@ def h(text)
 end
 
 get '/' do
-  @dataset = @memo.read
+  @contents = @memo.read
```

Comment:
```suggestion
  @memos = @memo.read
```

この場合はメモの配列=memosとした方がわかりやすいように思いますね。

https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
確かに具体的でわかりやすいですね。抽象度をコントロールする意識を持ちます。記事参考にさせていただきます！